components/empty-view/src/constants.ts

```diff
@@ -58,6 +61,7 @@ export const Illustration: Record<
   "not-found": bundleIllustrationSmart(NotFoundDark, NotFoundLight),
   settings: bundleIllustrationSmart(SettingsDark, SettingsLight),
   success: bundleIllustrationSmart(SuccessDark, SuccessLight),
+  media: bundleIllustrationSmart(MediaDark, MediaLight),
```

Comment:
It's not so important but can you put this after "general" to sort alphabetically? (Zeno team likes sorting :smile: )

Reply:
agree, my bad